development.md

```diff
@@ -10,15 +10,17 @@ docker compose watch
 
 * Now you can open your browser and interact with these URLs:
 
-Frontend, built with Docker, with routes handled based on the path: http://localhost:5173
-
 Backend, JSON based web API based on OpenAPI: http://localhost:8000
 
 Automatic interactive documentation with Swagger UI (from the OpenAPI backend): http://localhost:8000/docs
 
+Alternative interactive documentation (ReDoc): http://localhost:8000/redoc
+
 Adminer, database web administration: http://localhost:8080
 
-Traefik UI, to see how the routes are being handled by the proxy: http://localhost:8090
+RabbitMQ Management UI: http://localhost:15672
+
+Celery Flower (task monitoring): http://localhost:5555
```

Comment:
should we add steps to setup celery and rabbitMQ as well? with docker its pretty simple as `docker compose up` takes care of it but if someone wants to setup separately then there are no steps